apps/theming-designer/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "outDir": "lib",
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
```

Comment:
Can you add the following? These promote good practices and keep your project consistent with the others. (I'll also add this to the project template.)
```
"noImplicitAny": true,
"strictNullChecks": true,
```